### PR TITLE
(#1813) Remove USWDS Pointer Links

### LINKED ISSUE #1813: Remove Pointers to USWDS from NCIDS Doc Site Utility Pages
### Issue Description
There are some utility pages on the NCIDS Documentation Site that still point to and have text that refer the user to go to USWDS, and this content needs to be removed.

### Steps to reproduce the issue

1.  Go to https://designsystem-dev.cancer.gov/develop/foundations/layout-positioning/display
2. Look at the paragraph of text under the Utility Classes section
3. See that USWDS is still pointed to

### What's the expected result?
- There shall be no pointers to USWDS

### What's the actual result?
- There is indication for users to go to USWDS

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1813

## Pull Request Details

_Add description_

Closes #<TICKET-NUMBER>

## Author PR Checklist
Items that the author of the PR is responsible for checking before submitted the PR.

### General:
- [x] I have reviewed the acceptance criteria defined in the ticket and ensured the work has been completed.
- [x] The commit message passes [all quality commit message standards](https://github.com/NCIOCPL/ncids/wiki/Creating-Quality-Commit-Messages).
- [x] Unit tests have been updated or created to reflect any javascript changes.
- [x] Storybook scenarios have been updated or created to reflect any html/css/js changes.

### Accessibility:
- [x] WCAG 2.1 Level AA requirements have been met.

### Development:
- [x] Any new or updated javascript code has 100% unit test coverage.
- [x] New or updated breakpoints have regression images.
- [x] Breaking changes have been thoroughly documented in the PR.

## Product Reviewer PR Checklist
Items the product team is responsible for reviewing.

### General:
- [x] There are no unexpected or unapproved regression image changes. 
- [x] Functionality of interactive elements meet the acceptance criteria.
- [x] The product is visually and functionally the same across the different browsers.

### Accessibility:
- [x] AxeDev Tools: there are no new or outstanding accessibility issues introduced in this PR.
- [x] Lighthouse: scores have not noticeably decreased during this PR.
- [x] Wave: there are no new errors or contrast errors introduced in this PR.

## Design Reviewer PR Checklist
Items the design team is responsible for reviewing.  

### General:
- [ ] New or updated features introduced in this PR are developed mobile-first.
- [ ] Breakpoint changes and regression images match those breakpoints.
- [ ] This PR has been tested in all supported browsers at all breakpoints.

## Developer Reviewer PR Checklist
Items the development team is responsible for reviewing.

### General:
- [ ] New code passes code quality standards set by industry standards.
- [ ] The expected Storybook stories have been added or updated for the new or updated feature.
- [ ] The expected unit tests have been added or updated for the new or updated feature.

### Accessibility:
- [ ] VoiceOver: Described content matches with what was expected.
- [ ] Keyboard navigation: new or updated features and content are navigable via the keyboard.

### CODE REVIEW

**Review by olitharp-nci — CHANGES_REQUESTED** [review]
Clean up this PR. You have changes from other tickets here and also make sure you have no conflicts

File: docs/content/foundations/layout-positioning/float.mdx [hunk 1]
```diff
@@ -8,9 +8,7 @@ utility_modules:
   - name: Float
     description: Places an element on the left or right side of its container, allowing text and inline elements to wrap around it.
     utility_module_name: 'float'
-    utility_class_intro: |
-      The float utility classes have not been modified from the U.S. Web Design System (USWDS). Refer to the [USWDS Float utility page](https://designsystem.digital.gov/utilities/float/) for a complete listing of classes.
-    utility_class_display_component: 'FloatDemonstration'
+    utility_class_display_component: 'TestUtilityClassTable'
```
Comment (bryanpizzillo) [inline docs/content/foundations/layout-positioning/float.mdx:11]: @karlikpj why are you reverting this back to before there was a demonstration class/

File: docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js [hunk 2]
```diff
@@ -1,13 +1,19 @@
 export * as BoxModelDemonstration from './box-model-demonstation';
 export * as ClassValueDemonstration from './class-value-demonstration';
 export * as ColorDemonstration from './color-demonstration';
+export * as DisplayDemonstration from './display-demonstration';
```
Comment (bryanpizzillo) [inline docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js:4]: Why did this need to change? I am confused as these all should be working.

**Review by bryanpizzillo — DISMISSED** [review]

File: docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js (hunk 2 above)
Comment (karlikpj) [reply docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js:4]: they were just orded/alphabetized

File: docs/content/foundations/layout-positioning/float.mdx (hunk 1 above)
Comment (karlikpj) [reply docs/content/foundations/layout-positioning/float.mdx:11]: merge issue / fixing

**Review by sarinapadilla — APPROVED** [review]
The USWDS links have been removed from Float, List Reset, and Display. This looks good!

File: docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js
Comment (olitharp-nci) [inline docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js:1]: remove any changes from this file.

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**